configure.in

```diff
@@ -162,7 +162,9 @@ CITUSAC_PROG_CC_CFLAGS_OPT([-Wno-unused-parameter])
 CITUSAC_PROG_CC_CFLAGS_OPT([-Wno-sign-compare])
 CITUSAC_PROG_CC_CFLAGS_OPT([-Wno-missing-field-initializers])
 CITUSAC_PROG_CC_CFLAGS_OPT([-Wno-clobbered])
-CITUSAC_PROG_CC_CFLAGS_OPT([-Wno-gnu-variable-sized-type-not-at-end])
+if test x"$GCC" = x"no" ; then
```

Comment:
I'm not sure if this is needed anymore after Jelte's recent changes for flag check. If it is needed, we need to move it to a separate PR.